app/helpers/sessions_helper.rb

```diff
@@ -3,16 +3,41 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  def remember user
+    user.remember_token
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
   def current_user
```

Comment:
Hàm này có thực hiện những chức năng gì em?

Reply:
Xét xem id có đang trong session không, có thì set current_user chính là user đó, không thì log in